config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'upload/' => 'upload#index', as: 'upload'
```

Comment:
The reason for this is because this is how REST works. Have a look into REST. This might be helpful: http://code.tutsplus.com/tutorials/a-beginners-guide-to-http-and-rest--net-16340